### PR TITLE
React/DP-12932:  Storybook v5 upgrade.

### PR DESCRIPTION
**Pull request by smurrayatwork**
## Description
This upgrades storybook to version 5.0.x and adds the [storysource](https://github.com/storybooks/storybook/tree/v5.0.5/addons/storysource) and [a11y](https://github.com/storybooks/storybook/tree/v5.0.5/addons/a11y) addons.

## Related Issue / Ticket
- [JIRA issue](https://jira.mass.gov/browse/DP-12932)

## Steps to Test

1. Verify all automated tests pass and all stories work the same as before.

## Screenshots
screencapture-localhost-6006-2019-03-25-10_24_09

### CODE REVIEW

**Review by avrilpearl — APPROVED** [review]
LGTM! I updated the reference images and also moved the info button to the bottom right corner of the screen.

File: changelogs/DP-12932.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Major
+Added
+- (React) [*] DP-12932: Upgrades storybook and related add-ons to version 5.0.x.
```
Comment (clairesunstudio) [inline changelogs/DP-12932.md]:
```suggestion
- (React) [*] DP-12932: Upgrades storybook and related add-ons to version 5.0.x. #516
```

File: changelogs/DP-12932.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Major
+Added
+- (React) [*] DP-12932: Upgrades storybook and related add-ons to version 5.0.x. #516
+- (React) [*] DP-12932: Adds storysource and a11y add-ons.
```
Comment (clairesunstudio) [inline changelogs/DP-12932.md]:
```suggestion
- (React) [*] DP-12932: Adds storysource and a11y add-ons. #516 
```

File: react/backstop/backstop.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ module.exports = {
   report: ['browser', 'CI'],
   engine: 'puppeteer',
   engineOptions: {
-    args: ['--no-sandbox']
+    args: ['--no-sandbox', '--disable-dev-shm-usage']
```
Comment (clairesunstudio) [inline react/backstop/backstop.js]:
```suggestion
    args: ['--no-sandbox']
```

**Review by clairesunstudio — APPROVED** [review]

**Pull request merged**